editor/editor_help.cpp

```diff
@@ -915,13 +915,11 @@ void EditorHelp::_update_doc() {
 	// Properties overview
 	HashSet<String> skip_methods;
 
-	bool has_properties = cd.properties.size() != 0;
-	if (cd.is_script_doc) {
-		has_properties = false;
-		for (int i = 0; i < cd.properties.size(); i++) {
-			if (cd.properties[i].name.begins_with("_") && cd.properties[i].description.strip_edges().is_empty()) {
-				continue;
-			}
+	bool has_properties = false;
+	for (int i = 0; i < cd.properties.size(); i++) {
```

Comment:
I'd suggest using foreach syntax here instead:
```for (const DocData::PropertyDoc &prop : cd.properties)```
and then replacing `cd.properties[i]` with `prop`

It's more readable and compact

Reply:
agreed!